src/pandas_profiling/visualisation/plot.py

```diff
@@ -140,47 +140,64 @@ def get_correlation_font_size(n_labels) -> Optional[int]:
 
 
 @manage_matplotlib_context()
-def correlation_matrix(data: pd.DataFrame, vmin: int = -1) -> str:
+def correlation_matrix(
```

Comment:
This rendering logic should go in correlations.py

Reply:
Could someone please follow-up on getting the table format of the correlation in addition to the heatmap?  It would be very useful.